src/modules/powerrename/unittests/PowerRenameRegExTests.cpp

```diff
@@ -301,7 +301,7 @@ TEST_METHOD(VerifyReplaceFirstWildCardUseRegex)
     VerifyReplaceFirstWildcard(sreTable, ARRAYSIZE(sreTable), UseRegularExpressions);
 }
 
-TEST_METHOD(VerifyReplaceFirstWildCardUseRegexMatchAllOccurances)
+TEST_METHOD(VerifyReplaceFirstWildCardUseRegexMatchAllOccurrences)
```

Comment:
Can you open a PR with the fixes for `src/modules/powerrename` and `AccessibiltyChecker` (the last two files in this PR)?
Thanks.